src/components/ReferencePanel.js

```diff
@@ -25,7 +25,6 @@ class ReferencePanel extends React.Component {
         });
     }
     render (){
-        const layout = AutographaStore.layout;
```

Comment:
Why is this removed?

Reply:
Some variables are been removed since its declared and not used.